### PR TITLE
Ft update trainee dark mode

### PR DESCRIPTION
**Pull request by Gedeonir**
## What does this PR do?

this PR provides Dark mode of updating the details of trainee

## Description of Task to be completed?

1.  **Trainee_id** is required to get the trainee details
2. Other trainee's information can be updated except **Email cannot be updated, and trainee_id**

## How should this be manually tested?

- `Clone the repo`

- `checkout` to branch `ft-updateTraineeDarkMode`
- run git pull to have latest update

- Install all the required packages by running `npm i` or `npm install`

- Start app by `running npm run dev`
- navigate to list of trainee-applicant and choose trainee you want to edit by clicking three dot and choose edit. then you will be redirected to page to edit

- toogle the dark/light mode toggler to test

Dark
LIGHT

### CODE REVIEW

File: src/redux/actions/updateTrainee.ts [hunk 1]
```diff
@@ -69,9 +70,12 @@ export const getTraineeToUpdate = (Traineid: any) => async (dispatch: any) => {
 
 export const updateTraine = ({id,firstName,lastName,cycle_id}:any)=> async (dispatch: any) =>{
   try {
-    const datas = await axios.post('/',
+    const datas = await axios({
+      url: process.env.BACKEND_URL,
```
Comment (capitale250) [inline src/redux/actions/updateTrainee.ts]: we are using axiosconfig file

Comment (Gedeonir) [reply src/redux/actions/updateTrainee.ts]: Corrected

Comment (Gedeonir) [reply src/redux/actions/updateTrainee.ts]: solved

**Review by tysepa — APPROVED** [review]

File: src/redux/app/hook.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { TypedUseSelectorHook, useDispatch, useSelector } from "react-redux";
```
Comment (capitale250) [inline src/redux/app/hook.ts:1]: we have this hook arleady

**Pull request merged**